ics-frontend/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import {GalleryComponent} from './gallery/gallery.component';
+import { SingleImagePageComponent } from './single-image-page/single-image-page.component';
+import { ImageClassificationComponent } from './image-classification/image-classification.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path: 'gallery', component: GalleryComponent},
+  {path: 'imageClassification', component: ImageClassificationComponent},
```

Comment:
When you have multiple words in the URL path, it's a good practice to separate them with dashes (so in this case it would be `image-classification`)